### PR TITLE
More Ch1 notes

### PR DESCRIPTION
**Pull request by crazy4pi314**
Slight refactor with more notes I found, and better flowing exercises

### CODE REVIEW

File: 1-basics.md [hunk 1]
```diff
@@ -14,44 +14,66 @@ kernelspec:
...
   - Help: `?`
 - Status bar
-- Accessibility (color themes, screen readers, screencast mode, etc.)
-- VS Code terminal (xtermjs)
+- [Accessibility](https://code.visualstudio.com/docs/editor/accessibility) (color themes, screen readers, screencast mode, etc.)
+- [VS Code terminal](https://code.visualstudio.com/docs/terminal/advanced) (xtermjs)
 - [Navigating without a mouse](https://www.youtube.com/watch?v=dJWJ0hCAkAI)
 
-**Exercise**: clone a repo from GitHub and open in Codespaces (or your preferred platform). We will be working with it for the rest of the tutorial. E.g.: numpy (Python + C extensions, could be interesting for debugging), matplotlib, scipy.
+> **Exercise**: Explore VS Code acessability
```
Comment (ltalirz) [inline 1-basics.md:40]:
```suggestion
> **Exercise**: Explore VS Code acessibility
```

File: 1-basics.md [hunk 2]
```diff
@@ -14,44 +14,66 @@ kernelspec:
...
+> **Exercise**: Share your profile
+>
+> Either in your local or browser-based VS Code, try customizing some settings, themes, etc. either from a blank profile or one of the templates. Then export your current editor config as a gist (or locally if you prefer). If you have the URL for your profile gist, add it to the [discussion page](https://github.com/crazy4pi314/scipy-vscode-tutorial/discussions/17).
 
-- Python extension settings: formatters / linters
-  - available as extensions: pylint, flake8, black, autopep8, ...
+## Setting up for Python development
 
-**Customize your ride** - setting up our environment for Python development
+We will set up an existing Python project and install the necessary extensions. This piece is fairy quick and self-guided.
```
Comment (ltalirz) [inline 1-basics.md:67]: so, are we using an existing project or are we creating a new one (as suggested below?)

File: 1-basics.md [hunk 3]
```diff
@@ -14,44 +14,66 @@ kernelspec:
...
+  - Default intrepreter settings
+  - [Python environment manager](https://marketplace.visualstudio.com/items?itemName=donjayamanne.python-environment-manager)
+- Common Python extensions
+  - [Python extension pack by Don Jayamanne](https://marketplace.visualstudio.com/items?itemName=donjayamanne.python-extension-pack)
+  - [Jupyter](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter)
+  - formatters / linters: pylint, flake8, black, autopep8, ...
 
+> **Exercise**: New Python project from scratch
+>
+> In the Codespace for this workshop or locally (in a Dev Container) on your machine, Use VS Code to create a new virtual environment, install new packages with the `Python: Create Environment` command from the command pallet or the built-in terminal. Check that you have the intrepreter setup correclty by opening a new Jupyter notebook or script and import the package(s).
```
Comment (ltalirz) [inline 1-basics.md:79]: It's good to have an exploratory component, 
but let's also have some simple task people can check off to set a minimal level to which everyone should reach; something like

```python
import emoji
print(emoji.emojize("Meet your coding best friend: VS Code :red_heart:"))
```

**Comment by ltalirz** [comment]
So far, the exercises here are individual tasks.
Do we want to have a team component already in the first section, or do we start with this later?

File: 1-basics.md (hunk 2 above)
Comment (crazy4pi314) [reply 1-basics.md:67]: I think for this part, we should so folks can try their hand at the environment management? That dosen't preclued using pre-made ones later I think. Mostly I want to use the workshop devcontainer to control this experience.

File: 1-basics.md (hunk 3 above)
Comment (crazy4pi314) [reply 1-basics.md:79]: Yeah good idea

**Comment by ltalirz** [comment]
Here is a table you can use for the first point

 ```
## VS Code use over the years

From [Stack Overflow Annual Developer Surveys](https://insights.stackoverflow.com/survey)

| Year | VS Code [1] | Rank |
|------|-------------|------|
| 2016 | 7%          | 13   |
| 2017 | 23%         | ?    |
| 2018 | 35%         | 1    |
| 2019 | 51%         | 1    |
| 2020 | ?           |      |
| 2021 | 71%         | 1    |
| 2022 | 74%         | 1    |
| 2023 | 74%         | 1    |

[1] Percentage of respondents using VS Code (multiple answers possible).
```

**Comment by ltalirz** [comment]
Some more statistics for point 1
* #1 repository on GitHub by number of contributors ([2022 State of the Octoverse](https://marketplace.visualstudio.com/search?target=VSCode&category=All%20categories&sortBy=Installs)
* [49k extensions on the Marketplace](https://marketplace.visualstudio.com/search?target=VSCode&category=All%20categories&sortBy=Installs)

**Review by ltalirz — APPROVED** [review]

**Pull request merged**